### PR TITLE
doc: update docs to add `receipt_header` field

### PR DESCRIPTION
**Pull request by taylrj**

### CODE REVIEW

File: doc/periodic-donation.apib [hunk 1]
```diff
@@ -26,6 +26,7 @@ The Periodic Donation resource has the following attributes;
 - cardholder.national_id
 - cardholder.phone_number
 - cardholder.zip_code
+- cardholder.receipt_header
```
Comment (babygoat) [inline doc/periodic-donation.apib:29]: We should promote the `receipt_header` to the top-level fields.

**Comment by babygoat** [comment]
Can we have the donors leave blank to their receipt header?

**Comment by taylrj** [comment]
Yes, receipt header is only required when donors need to receive receipt.

File: doc/periodic-donation.apib (hunk 1 above)
Comment (taylrj) [reply doc/periodic-donation.apib]: Updated.

File: doc/periodic-donation.apib [hunk 2]
```diff
@@ -138,13 +140,14 @@ The states *id* and *order_number* are assigned by the TWReporter Go API at the
                         "name": "name(string) is optional",
                         "zip_code": "zip_code(string) is optional",
                         "phone_number": "phone_number(string) is optional",
-                        "national_id": "national_id(string) is optional"
+                        "national_id": "national_id(string) is optional",
```
Comment (babygoat) [inline doc/periodic-donation.apib:143]: Is trailing comma a valid json format?

Comment (taylrj) [reply doc/periodic-donation.apib]: No, it isn't. Next commit fixes the format.

**Comment by babygoat** [comment]
LGTM

**Pull request merged**